lib/index.js

```diff
@@ -163,6 +163,11 @@ internals.handler = function (route, handlerOptions) {
 
         const promise = settings.httpClient.request(request.method, uri, options);
 
+        request.raw.req.once('aborted', () => {
+
+            promise.req.abort();
```

Comment:
`abort` is deprecated in favor of `destroy`, but `hapi`'s documentation and examples are all using `abort`. Which one do you prefer?